### PR TITLE
Readme build instructions

### PR DESCRIPTION
**Pull request by AOrobator**

### CODE REVIEW

**Comment by atticoos** [comment]
Can we break this PR up? 

Build instructions would be great to get in, refactoring should be held on a different branch

**Comment by AOrobator** [comment]
Just broke it up. I'll make a PR for refactoring as well, but it has dependencies on this branch so I'll wait

File: app/src/main/java/com/spielpark/steve/bernieapp/fragments/NavigationDrawerFragment.java [hunk 1]
```diff
@@ -241,17 +241,6 @@ public void onConfigurationChanged(Configuration newConfig) {
     }
 
     @Override
-    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-        // If the drawer is open, show the global app actions in the action bar. See also
-        // showGlobalContextActionBar, which controls the top-left area of the action bar.
-        if (mDrawerLayout != null && isDrawerOpen()) {
-            inflater.inflate(R.menu.global, menu);
-            showGlobalContextActionBar();
-        }
```
Comment (atticoos) [inline app/src/main/java/com/spielpark/steve/bernieapp/fragments/NavigationDrawerFragment.java:250]: Intentional to exclude this edge?

Comment (AOrobator) [inline app/src/main/java/com/spielpark/steve/bernieapp/fragments/NavigationDrawerFragment.java:250]: I added it back in a later commit. It's in the refactoring PR I'm pretty sure. I just wanted to remove the menu when the nav drawer popped up. Should've just deleted L248

Comment (atticoos) [inline app/src/main/java/com/spielpark/steve/bernieapp/fragments/NavigationDrawerFragment.java:250]: Is this going to be broken w/o #7?

Comment (AOrobator) [inline app/src/main/java/com/spielpark/steve/bernieapp/fragments/NavigationDrawerFragment.java:250]: Nope, should work fine. Only thing is the app name won't change when the drawer is opened.

**Pull request merged**